src/react/Title.css

```diff
@@ -0,0 +1,46 @@
+.message-container {
+    position: fixed;
+    inset: 0;
+    z-index: -1;
+    pointer-events: none;
+    white-space: nowrap;
+}
+
+.titleScreen {
+    position: fixed;
+    top: 50%;
+    left: 0;
+    transform: translateY(-60%);
```

Comment:
when there is only a title (no subtitle) it should be in the same position so **crosshair is visible**:
<img width="581" alt="image" src="https://github.com/zardoy/prismarine-web-client/assets/46503702/b0ef7b8b-347d-4751-b3f9-1187e6738c17">

Reply:
its not related to the current line of code btw